### PR TITLE
Changing Bigtable stub factories to use beta stubs.

### PR DESCRIPTION
**Pull request by dhermes**
@tseaver Sorry I missed this in #1245. After this I'll get back to actually bringing features over.

### CODE REVIEW

File: gcloud/bigtable/_helpers.py [hunk 1]
```diff
@@ -85,8 +90,11 @@ def make_stub(client, stub_factory, host, port):
...
+    client_credentials = implementations.ssl_client_credentials(
+        root_certificates, private_key=None, certificate_chain=None)
+    channel = implementations.secure_channel(
+        host, port, client_credentials)
     custom_metadata_transformer = MetadataTransformer(client)
-    return stub_factory(host, port,
-                        metadata_transformer=custom_metadata_transformer,
-                        secure=True,
-                        root_certificates=get_certs())
+    return stub_factory(channel, host=host,
```
Comment (tseaver) [inline gcloud/bigtable/_helpers.py]: Prolly a `grpc` issue, but why is `host` included here?  It should be hard-wired into `channel` already, no?

Comment (dhermes) [inline gcloud/bigtable/_helpers.py]: Yes, you saw that right. I don't know why the redundancy is there. Maybe the second `host` is optional and I don't need to be using it?

Comment (dhermes) [inline gcloud/bigtable/_helpers.py]: OK I just removed `host` here and re-ran the system tests in `gcloud-python-bigtable` with no issues. It must be intended as an optional over-ride for the `channel`.

**Comment by dhermes** [comment]
@tseaver PTAL I updated to drop the use of `host` in the stub factories.

**Comment by tseaver** [comment]
LGTM

**Pull request merged**